lib/explorer/backend/lazy_series.ex

```diff
@@ -299,6 +375,14 @@ defmodule Explorer.Backend.LazySeries do
     Backend.Series.new(data, :boolean)
   end
 
+  @impl true
+  def all_equal?(%Series{} = left, %Series{} = right) do
```

Comment:
Shouldn’t this be `is_all_equal`?

Reply:
I renamed to `Series.all_equal/2`, without the `?`.